pkgs/development/tools/gotags/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "gotags";
+  version = "1.4.1";
+
+  goPackagePath = "github.com/jstemmer/gotags";
+
+  src = fetchFromGitHub {
+    owner = "jstemmer";
+    repo = pname;
+    rev = "4c0c4330071a994fbdfdff68f412d768fbcca313";
+    sha256 = "sha256-cHTgt+zW6S6NDWBE6NxSXNPdn84CLD8WmqBe+uXN8sA=";
+  };
+
+  meta = with lib; {
+    description = "ctags-compatible tag generator for Go";
+    homepage = "https://${goPackagePath}";
```

Comment:
```suggestion
  src = fetchFromGitHub {
    owner = "jstemmer";
    repo = pname;
    rev = "4c0c4330071a994fbdfdff68f412d768fbcca313";
    sha256 = "sha256-cHTgt+zW6S6NDWBE6NxSXNPdn84CLD8WmqBe+uXN8sA=";
  };

  goPackagePath = "github.com/jstemmer/gotags";

  meta = with lib; {
    description = "ctags-compatible tag generator for Go";
    homepage = "https://github.com/jstemmer/gotags";
```

Reply:
happy to move `goPackagePath` down, but is it critical that we repeat ourselves?

Reply:
Yes, meta.homepage should be clickable to make reviewer lives easier.

Reply:
sorry for the delay, I rebased and everything should be done

Reply:
No, please keep it simple. The homepage could also be something else like the docs or website.

Reply:
sorry, looks like it is just the repo; I am mostly mirroring the old derivation, and linking to pkg.go.dev seems like bad form.